### PR TITLE
samples: add sample code to query regional Dialogflow agent

### PR DESCRIPTION
**Pull request by martini9393**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-dialogflow/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Google LLC
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java:2]:
```suggestion
 * Copyright 2020 Google LLC
```

File: samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dialogflow;
+
+// [START dialogflow_detect_intent_text]
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java:19]: This should be unique (it looks like there's another sample with this same snippet id).

**Review by chingor13 — CHANGES_REQUESTED** [review]
LGTM with a few minor changes

File: samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java (hunk 2 above)
Comment (martini9393) [reply samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java]: Updated snippet id, PATL, thanks!

File: samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            queryResult.getIntent().getDisplayName(), queryResult.getIntentDetectionConfidence());
+        System.out.format("Fulfillment Text: '%s'\n", queryResult.getFulfillmentText());
+
+        queryResults.put(text, queryResult);
+      }
+    }
+    return queryResults;
+  }
+}
+// [END dialogflow_detect_intent_text]
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java:79]: This one needs to match the above snippet of too

Comment (martini9393) [reply samples/snippets/src/main/java/com/example/dialogflow/DetectIntentWithLocation.java]: done.

**Review by chingor13 — APPROVED** [review]

**Pull request merged**